FunctionalEquivalence/subworkflows/F1Evaluation.wdl

```diff
@@ -235,52 +218,55 @@ def diff_plot(ax, data, dataset, var_type, region, tool1_label, tool2_label, sig
     else:
         print('Invalid plot data for {} {} {}'.format(dataset, var_type, region))
```

Comment:
This and the other `.format`'s could probably be made for readable using f-strings, if you want ("while you're here"). 

E.g. At this line, you can use
`print(f'Invalid plot data for {dataset} {var_type} {region}')`